choc-pizza.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Chocolate Pizza
+		</title>
+		<link href="reset-choc.css" type="text/css" rel="stylesheet" />
+		<link href="style-choc.css" type="text/css" rel="stylesheet" />
+	</head>
+	<body>
+		<header>
+			<div id="delicious">
+				Delicious
+			</div>
+			<div id="sub-delicious">
+				THE BEST FOOD BLOG ON THE WEB
+			</div>
+			<div id="top-logo">
+				<img src="https://www.dropbox.com/s/82h4pg4ovo5k0mx/logo.png?dl=0" />
+			</div>
+		</header>
+		<aside id="top-right">
+			<img id="facebook" src="https://www.dropbox.com/s/ry78j6lphblmajv/fb-icon.png?dl=0" />
+			<img id="twitter" src="https://www.dropbox.com/s/39hha9uzrsm34o6/twit-icon.png?dl=0" />
+			<img id="google-plus" src="https://www.dropbox.com/s/7vya1cra1op5anu/gp-icon.png?dl=0" />
+			<img id="instagram" src="https://www.dropbox.com/s/izzcep2mviaxw3y/insta-icon.png?dl=0" />
+			<img id="flickr" src="https://www.dropbox.com/s/wzsgnq59bahvkzz/flic-icon.png?dl=0" />
+			<img id="pinterest" src="https://www.dropbox.com/s/8k82f2ndd9fu51p/pint-icon.png?dl=0" />
+			<img id="rss" src="https://www.dropbox.com/s/ehx10qs6gp8ivd1/rss-icon.png?dl=0" />
+			<img id="email" src="https://www.dropbox.com/s/5hgigjidh8464jf/mail-icon.png?dl=0" />
+		</aside>
+		<div class="slash-border">
+		</div>
+		<header>
+			<div id="chocolate-pizza">Chocolate Pizza
+			</div>
+			<div id="sub-chocolate-pizza">SUB CHOCOLATE PIZZA
+			</div>
+		</header>
+		<aside id="print"> 
```

Comment:
Review the use of the aside tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside